bin/buckinghamshire/create-parish-bodies

```diff
@@ -0,0 +1,90 @@
+
+use strict;
+use warnings;
+
+BEGIN {    # set all the paths to the perl code
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../setenv.pl";
+}
+
+use mySociety::MaPit;
+use FixMyStreet::DB;
+
+# Mapping for parishes that aren't called "$name Parish Council"
+# The key is the mapit name, value is the desired display name.
+# Anything not in this list will default to "$mapit_name Parish Council"
+my $name_mappings = {
+    'Addington' => 'Addington (meeting)',
+    'Amersham' => 'Amersham Town Council',
+    'Aston Sandford' => 'Aston Sandford (meeting)',
+    'Aylesbury' => 'Aylesbury Town Council',
+    'Barton Hartshorn' => 'Barton Hartshorn (meeting)',
+    'Beaconsfield' => 'Beaconsfield Town Council',
+    'Biddlesden' => 'Biddlesden (meeting)',
+    'Bledlow-cum-Saunderton' => 'Bledlow-Cum-Saunderton Parish Council',
+    'Buckingham' => 'Buckingham Town Council',
+    'Chesham' => 'Chesham Town Council',
+    'Chetwode' => 'Chetwode (meeting)',
+    'Cholesbury-cum-St Leonards' => 'Cholesbury-Cum-St Leonards Parish Council',
+    'Coldharbour' => 'Coldharbour',
+    'Creslow' => 'Creslow (meeting)',
+    'Dinton-with-Ford and Upton' => 'Dinton-with-Ford and Upton',
+    'Dorton' => 'Dorton Parish Council (meeting)',
+    'Drayton Beauchamp' => 'Drayton Beauchamp (meeting)',
+    'Dunton' => 'Dunton (meeting)',
+    'Edlesborough' => 'Edlesborough Northall and Dagnall Parish Council',
+    'Gerrards Cross' => 'Gerrards Cross Town Council',
+    'Hedsor' => 'Hedsor (Meeting)',
+    'Hoggeston' => 'Hoggeston (meeting)',
+    'Kingsey' => 'Kingsey (meeting)',
+    'Kingswood' => 'Kingswood (meeting)',
+    'Longwick-cum-Ilmer' => 'Longwick cum Ilmer Parish Council',
+    'Marlow' => 'Marlow Town Council',
+    'Nether Winchendon' => 'Nether Winchendon (meeting)',
+    'Poundon' => 'Poundon (meeting)',
+    'Princes Risborough' => 'Princes Risborough Town Council',
+    'Shalstone' => 'Shalstone (meeting)',
+    'Thornton' => 'Thornton (meeting)',
+    'Water Stratford' => 'Water Stratford (meeting)',
+    'Wingrave with Rowsham' => 'Wingrave-with-Rowsham Parish Council',
+    'Winslow' => 'Winslow Town Council',
+    'Wooburn' => 'Wooburn and Bourne End Parish Council',
+    'Woodham' => 'Woodham (meeting)',
+    'Wotton Underwood' => 'Wotton Underwood (meeting)',
+};
+
+my $parent_id = "163793"; # Buckinghamshire Council Unitary Authority on Mapit
+
+# Fetch the list of parishes from MapIt
+my $mapit_areas = mySociety::MaPit::call('area', "$parent_id/children", type => 'CPC');
+my @areas = values %$mapit_areas;
+
+my $db = FixMyStreet::DB->schema->storage;
+$db->txn_do(sub {
+    foreach my $area (@areas) {
+        my $id = $area->{id};
+        my $name = $name_mappings->{$area->{name}} || "$area->{name} Parish Council";
+
+        my $body_rs = FixMyStreet::DB->resultset("Body")->search({
+            id => $id,
```

Comment:
Whilst we historically set the ID of bodies to the area ID, the code was generalised to not require this some time ago and we shouldn't do it any longer.

Reply:
Removed this line in 1dfd1c482001296dae538d493f0c9bb85ec4a76a